### PR TITLE
Update docker image

### PR DESCRIPTION
**Pull request by enolfc**
# Summary

---

**Related issue :**

### CODE REVIEW

**Comment by enolfc** [comment]
Failing because the VM image for building is not found :(, will try to fix that and rebuild

File: ubuntu/appdb/docker.yaml [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
   notes: |
-    Upgrade to docker 25.0.4 and docker-compose 2.24.7
-    Upgrade kubernetes components to 1.28.7-1.1
-  url: https://api.cloud.ifca.es:8080/swift/v1/egi_endorsed_vas/Docker.Ubuntu.22.04-2024.03.14.ova
-  sha512: "011a03a64dce27db4a213fdbdefcbcaadbe423335312ac32a0cead5ba79aba440603cbe8cfec70a50daa3bc6d48e9befdbb86f392b28b3e6f5d87cf5c64553c3"
+    Upgrade docker to 27.2.1
+    Upgrade docker-compose to 2.29.3
+    Upgrade kubernetes components to 1.30.5-1.1
+  url: https://api.cloud.ifca.es:8080/swift/v1/egi_endorsed_vas/Docker.Ubuntu.22.04-2024.09.13.qcow2
+  sha512: "9e19ff5c80a646c173b94bfe7401a38ed7fc87b7617edfea9183cdfc9dc3e9ab51258170cf51fd807d3a20ebf7bf6faa151a8dcabfb4d44d66bebd7d29d0121d"
```
Comment (sebastian-luna-valero) [inline ubuntu/appdb/docker.yaml:10]: Could you please double check the checksum?

**Review by sebastian-luna-valero — CHANGES_REQUESTED** [review]

**Comment by enolfc** [comment]
Everytime the image is built the checksum changes, so we cannot update both the yaml and the image in the same PR?

**Comment by enolfc** [comment]
Forcing merge for now as the previous image expires today.
We need to have a better way of handling the shasum

**Pull request merged**